src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java

```diff
@@ -470,11 +470,13 @@ public Projectile get(final EntityDamageByEntityEvent e) {
 			}
 		}, 0);
 		// EntityDeathEvent
-		EventValues.registerEventValue(EntityDeathEvent.class, ItemStack[].class, new Getter<ItemStack[], EntityDeathEvent>() {
+		EventValues.registerEventValue(EntityDeathEvent.class, ItemType[].class, new Getter<ItemType[], EntityDeathEvent>() {
```

Comment:
This shouldn't be changing the return type to ItemType. Keep it as ItemStack, ItemType is for literals. ItemStacks provide more information.

Reply:
As you can see here 
![image](https://user-images.githubusercontent.com/98935832/209402232-dd9538da-d850-4960-9bcc-d5b766e5360b.png)
There's no plural for the ItemStack classinfo so you can't get the plural event value for that class.

ItemType on the other hand
![image](https://user-images.githubusercontent.com/98935832/209402336-d89bb660-19a0-4fac-846c-102fca69214f.png)
does have the plural form of `items` and `materials` unlike ItemStack which only has the singular form

Reply:
Well that's an issue we need to solve now. We can't be returning ItemType, especially in this popular event.

I suggest that items be associated with ItemStack as that's what is the main ClassInfo we should be using. ItemType is for literals. Materials is fine under ItemType but should have material single too.

Item and Items for ItemStack material and Item type for ItemType.

@APickledWalrus thoughts?